packages/gestalt/src/Table.css

```diff
@@ -14,6 +16,15 @@
   composes: paddingX3 from "./boxWhitespace.css";
 }
 
-.thead {
-  composes: borderBottom from "./Borders.css";
+.trClickable:hover {
+  background-color: #efefef;
```

Comment:
@PayalModi could you also add `composes: accessibilityOutline from "./Focus.css";` so we use the same focus style here as all of our components?